### PR TITLE
jsonify the spreadsheet as part of the site deploy process

### PR DESCRIPTION
**Pull request by rfreebern**
reducing external dependencies by fetching the google sheet data and jsonifying it as part of the deploy process, rather than relying on the tigg appspot service.

### CODE REVIEW

**Comment by elidickinson** [comment]
Ohh fancy. I can’t test it right now but that looks about right. Node’s got a package for everything huh

I fully endorse having zero infrastructure reliant on me

File: scripts/fetch-json.js [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+    let aTime = DateTime.fromISO(a.datetime).toSeconds();
+    let bTime = DateTime.fromISO(b.datetime).toSeconds();
+    return bTime - aTime;
+  });
+
+  fs.writeFileSync('content.json', JSON.stringify(entries, null, 2));
+});
+
+function dateTimeFromSheetsDateTime(sheetsDateTime) {
+  const numbers = Array.from(sheetsDateTime.matchAll(/\d+/g)).flat(); // Extract numbers
```
Comment (ocean) [inline scripts/fetch-json.js:53]: Nice! 👍

**Review by ocean — APPROVED** [review]
LGTM

**Review by dougmakingstuff — APPROVED** [review]
Nice!

**Pull request merged**